src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk

```diff
@@ -26,30 +28,48 @@ Before you create the new detailed guide or add the form pattern to an existing
 If it is, do not repeat it. Add a link and journey to that content instead.
 </p>
 
+<h3 class="govuk-heading-m" id="Detailed examples">Types of forms</h3>
+
 <p class="govuk-body">
 Types of form that you can use in a detailed guide include the following: 
 </p>
 
 <ul class="govuk-list govuk-list--bullet">
-<li>Online services — These are built by service teams on the tax platform behind the Government Gateway, accessed with a green button with a URL, starting with https://www.tax.service.gov.uk</li>
-<li>Interactive guidance (IG) forms — these are added as external links. The URL will begin with https://www.tax.service.gov.uk/guidance</li>
-<li>Interactive PDFs that can be completed on-screen — these are added as file attachments and require Adobe to open and complete</li>
-<li>PDFs that cannot be completed on screen — these are added as file attachments and do not require Adobe to open</li>
-<li>LC forms — these are added as external links. The URL will begin with https://public-online.hmrc.gov.uk/lc/content</li>
-<li>G-forms — these are added as external links. The URL will begin with https://tax.service.gov.uk</li>
-<li>forms in open document formats — forms in .ods or odt. format are added as file attachments</li>
-<li>KANA forms — these are added as external links. The URL will begin with https://online.hmrc.gov.uk/shortforms</li>
+<li><span class="govuk-!-font-weight-bold">Online services</span> — these are built by service teams on the tax platform behind the Government Gateway, accessed with a green button. The URL, starts with ‘https://www.taxservices.gov.uk/’ often followed by ‘/digital-forms/form/form-name’.</li>
```

Comment:
The `<span class="govuk-!-font-weight-bold">` needs to be done for all the `<strong>` tags